sw/airborne/firmwares/rotorcraft/rotorcraft_datalink.c

```diff
@@ -94,6 +94,20 @@ void firmware_parse_msg(void)
           /* local NED with x/y/z as velocity and yaw as absolute angle */
           autopilot_guided_move_ned(x, y, z, yaw);
           break;
+        case 0x71:
+          /* local NED with x/y/z as velocity and yaw as absolute angle */
+          autopilot_guided_set_vertical_velocity(z);
+          break;
+        case 0x72:
```

Comment:
what's the difference with 0x71 and 0x72, it also doesn't seem to be just in python script